lib/PrintAsClang/DeclAndTypePrinter.cpp

```diff
@@ -1478,6 +1478,10 @@ class DeclAndTypePrinter::Implementation
       specialNames[{ID_CoreGraphics, ctx.getIdentifier("CGFloat")}]
         = { "CGFloat", false };
 
+      Identifier ID_CoreFoundation = ctx.getIdentifier("CoreFoundation");
+      specialNames[{ID_CoreFoundation, ctx.getIdentifier("CGFloat")}]
```

Comment:
`ctx.Id_CoreFoundation` exists; you can use that instead.

(For that matter, so do `ctx.ID_CoreGraphics` and `ctx.Id_CGFloat`, so *all* of the `ctx.getIdentifier()` calls relating to CGFloat are unnecessary. I think there's been a little bit too much copy-and-paste programming...)

Reply:
Thanks. I'll update those as well.